### PR TITLE
Improve the documentation of the reoptimise argument

### PR DESCRIPTION
**Pull request by bkhelifi**
**Description**

This simple PR describes a bit more the behaviour of this argument.

### CODE REVIEW

**Review by MRegeard — APPROVED** [review]
Thanks @bkhelifi,
No comment on my side.

**Comment by MRegeard** [comment]
> Thanks @bkhelifi ! This is indeed a common query.
> 
> I think the `FluxPointsEstimator` needs to be adapted, rather than the `FluxEstimator`. It is also useful to specify that the spectral parameters are always kept frozen.
> 
> Probably how the estimators work can go in the documentation, and the doc strings can refer to the said section

Hi @AtreyeeS,

Why not also do this on the `FluxEstimator` ? It the same thing for this object right ? Even though it's not public, for homogeneity we could repeat this everywhere ?

**Comment by bkhelifi** [comment]
I have implemented the comments of @AtreyeeS.

File: gammapy/estimators/points/lightcurve.py [hunk 1]
```diff
@@ -87,6 +89,9 @@ def __init__(self, time_intervals=None, atol="1e-6 s", **kwargs):
         self.time_intervals = time_intervals
         self.atol = u.Quantity(atol)
 
+        if "reoptimize" not in kwargs:
+            kwargs["reoptimize"] = False
+
```
Comment (QRemy) [inline gammapy/estimators/points/lightcurve.py:94]:
```suggestion
```

the default is already False as it is inherited from the `FluxEstimator`

**Review by registerrier — APPROVED** [review]
Thanks @bkhelifi . No comments from my side.

**Pull request merged**